Connectors/Autocad/Speckle.Connectors.AutocadShared/Speckle.Connectors.AutocadShared.shproj

```diff
@@ -3,6 +3,15 @@
     <PropertyGroup Label="Globals">
         <ProjectGuid>{41BC679F-887F-44CF-971D-A5502EE87DB0}</ProjectGuid>
     </PropertyGroup>
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
```

Comment:
idk if this change is important, but rider said I had "invalid build configurations" and offered to automatically fix it for me